FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/defender_config.h

```diff
@@ -52,4 +52,44 @@
 
 /************ End of logging configuration ****************/
 
+/**
+ * AWS IoT Device Defender Service supports both long and short names for keys
+ * in the report sent by a device. For example,
+ *
+ * A device defender report using long key names:
```

Comment:
See previous comment about capitalizing "Device Defender."

Reply:
Have updated all instances to be "**D**~~d~~evice **D**~~d~~efender"